resources/public/api/conf/1.0/application.raml

```diff
@@ -890,6 +921,13 @@ types:
                                       "code" : "BONUS_CLAIM_ALREADY_EXISTS",
                                       "message" : "The investor’s bonus payment has already been requested"
                                     }
+                                bonusRequestAlreadySuperseded:
+                                  description: The 'supersededTransactionID' and 'Superseded Transaction Amount' in the request match to a transaction ID and corresponding 'bonusDueForPeriod' amount on an existing transaction record for this account.
```

Comment:
'Superseded Transaction Amount' is not a json field name. Should match the field name in the json as per 'bonusDueForPeriod'